app/src/main/java/com/anytypeio/anytype/ui/onboarding/OnboardingFragment.kt

```diff
@@ -139,10 +136,9 @@ class OnboardingFragment : Fragment() {
     }
 
     @Composable
-    @OptIn(ExperimentalAnimationApi::class)
     private fun OnboardingScreen() {
         MaterialTheme {
-            val navController = rememberAnimatedNavController()
+            val navController = rememberNavController()
```

Comment:
does slide animation still work?

Reply:
checked, everything is working as it should be